src/database/index.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+const db = `mongodb+srv://${process.env.DATABASE_USER}:${process.env.DATABASE_PASSWORD}@cluster0.t3hjt.mongodb.net/${process.env.DATABASE_NAME}?retryWrites=true&w=majority`
+
+
+const databaseConnection = {
+    getConnect: () => {
+        mongoose
```

Comment:
Codacy has a fix for the issue: [Replace `········` with `····`](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7801432)
```suggestion
    mongoose
```